hedera-mirror-web3/src/main/java/com/hedera/mirror/web3/repository/properties/CacheProperties.java

```diff
@@ -0,0 +1,15 @@
+package com.hedera.mirror.web3.repository.properties;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.validation.annotation.Validated;
+
+@Data
+@Validated
+@ConfigurationProperties(prefix = "hedera.mirror.web3.cache")
+public class CacheProperties {
+
+    private long stateCacheSeconds = 5;
+    private long entityCacheSeconds = 30;
```

Comment:
As I noted both in the issue and in my feedback, it should be a caffeine cache string to allow for full customization. Here it's only allowing the seconds to be tweaked, but not the max items or expire after write, etc.

Also, it's redundant to have cache twice in the name of `hedera.mirror.web3.cache.stateCacheSeconds`.

```suggestion
    @NotBlank
    private String contractState = "expireAfterWrite=5s,maximumSize=10000,recordStats";

    @NotBlank
    private String entity = "expireAfterWrite=30s,maximumSize=10000,recordStats";
```